.github/workflows/split-monorepo.yml

```diff
@@ -18,10 +18,6 @@ jobs:
     - name: Split repository into git mirrors
       env:
         SSH_AUTH_SOCK: /tmp/ssh_agent.sock
-        METRICS_SDK_NODE_PRIVATE_KEY: ${{ secrets.METRICS_SDK_NODE_PRIVATE_KEY }}
```

Comment:
Can you remove these secrets from the git repo and 1password when this gets merged?

Reply:
yep